clang/lib/DPCT/GenMakefile.cpp

```diff
@@ -153,9 +153,37 @@ static void getCompileInfo(
     std::string NewOptions;
     bool IsObjName = false;
     bool IsObjSpecified = false;
+
+    // -isystem
+    bool ISISystem = false;
+
     const std::string Directory = Entry.second[0];
     std::unordered_set<std::string> DuplicateDuplicateFilter;
     for (const auto &Option : Entry.second) {
+
+      if (ISISystem) {
+        ISISystem = false;
+        std::string IncPath = Option;
+
+        if (!llvm::sys::fs::exists(IncPath)) {
+          // Skip including path that does not exist.
```

Comment:
for user, migration machine may not be the build machine,  so maybe it's not good to remove the non exist inlucde in migration machine.

Reply:
OK

Reply:
done